snapshot_manager/snapshot_manager/github_util.py

```diff
@@ -377,36 +390,30 @@ def remove_labels_safe(
             logging.info(f"Removing label '{label}' from issue: {issue.title}")
             issue.remove_from_labels(label)
 
-    @typing.overload
-    def minimize_comment_as_outdated(
-        self, comment: github.IssueComment.IssueComment
-    ) -> bool: ...
-
-    @typing.overload
-    def minimize_comment_as_outdated(self, node_id: str) -> bool: ...
-
     def minimize_comment_as_outdated(
         self,
-        object: str | github.IssueComment.IssueComment,
+        issue_comment_or_node_id: github.IssueComment.IssueComment | str,
     ) -> bool:
```

Comment:
Having just one function instead of the overloads makes more sense if the function only accepts a single argument.